modules/MQTTBridge/src/mqttbridge.cpp

```diff
@@ -7,19 +7,20 @@
 #include "mqtt.hpp"
 #include <random>
 
+
 using namespace ROSChannels;
 
 using std::placeholders::_1;
 
 MqttBridge::MqttBridge() : Module(MqttBridge::moduleName),
 						   v2xMsgSub(*this, std::bind(&MqttBridge::onV2xMsg, this, _1))
 {
-	declare_parameter("broker_ip");
-	declare_parameter("pub_client_id");
-	declare_parameter("username");
-	declare_parameter("password");
-	declare_parameter("topic");
-	declare_parameter("quality_of_service");
+	declare_parameter("broker_ip","192.71.100.198");
```

Comment:
Hard coding an IP-address?

Reply:
Same here, a value is required when a parameter is declared. Could be localhost or something to clarify that it is something to be changed. On the other hand for us this is the broker adress and we should not use another.